TodoList/Tests/TodoList.Api.Tests/Controllers/ItemsControllerTest.cs

```diff
@@ -42,8 +50,10 @@ public void SetUp()
         [Test]
         public async Task GetAsync_WithoutParams_ReturnsAllItems()
         {
+            _repository.GetAllItemsAsync().Returns(s_items);
+
             var contentResult = await _controller
-                .ExecuteAction(controller => controller.GetAsync());
+                .ExecuteAction(controller => controller.GetItemsAsync());
             contentResult.TryGetContentValue<Item[]>(out var items);
```

Comment:
Your tests should not be dependant on controller implementation.
```suggestion
             contentResult.TryGetContentValue<IEnumerable<Item>>(out var items);
```

Reply:
same as above 7d38c9c